docs/UserGuide.md

```diff
@@ -101,30 +102,32 @@ If you type in a command with no arguments and the command requires arguments, t
 
 Format: `help`
 
-#### _Clearing all entries_ :
+#### _Clearing all entries_
 
 Clears all entries from Notor. Be warned; data will be deleted and **will not be saved**. The intended use of this
  command is to clear the dummy starting data, but you can also use it to reset your Notor from within the program.
 
 Format: `clear`
 * Take care not to confuse this command with the more specific`/clearnote` command.
 
-#### _Exiting the program_ :
+#### _Exiting the program_
 
 Exits the program and saves your data.
 
 Format: `exit`
 
-####<a name = "exporting-data"></a> _Exporting data_ :
+<a name = "exporting-data"></a> 
+####_Exporting data_
```

Comment:
There is a bug with this due to the lack of space between #### and _

Reply:
Not fixed yet

Reply:
got it